component/authenticator/src/webapp/voiceOtp.jsp

```diff
@@ -0,0 +1,259 @@
+<%--
```

Comment:
Add the correct license header with the appropriate year. You can refer to it here. https://github.com/wso2/identity-apps/blob/35af68035b1235dd420116e691de8db02bdb1292/identity-apps-core/apps/x509-certificate-authentication-portal/src/main/webapp/tenant-resolve.jsp#L1-L17